website/docs/documentationFile.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Take a look at the other files in the folder for API references (https://github.com/premake/premake-core/blob/master/website/docs/includedirs.md?plain=1).

Typical format is:

Description

Example API Usage

Applies To

Availability

Examples

See Also (Put links out to MSFT docs here)